<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AVSGenerator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imgs.qrc">
    <normaloff>:/Icones/icone.ico</normaloff>:/Icones/icone.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="grupoArquivos">
      <property name="title">
       <string>Arquivos</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="4">
        <widget class="QToolButton" name="botaoSaida">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="caixaVideo">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="textoSaida">
         <property name="text">
          <string>Saída:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QToolButton" name="botaoVideo">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="caixaSaida">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="textoVideo">
         <property name="text">
          <string>Vídeo:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="grupoOpcoes">
      <property name="title">
       <string>Opções</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QPushButton" name="botaoEdComentarios">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Editar Comentários</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="botaoEdVideo">
         <property name="text">
          <string>Editar Vídeo</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="botaoCorte">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Corte</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="botaoEdLegendas">
         <property name="text">
          <string>Editar Legendas</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" rowspan="2">
        <widget class="QPushButton" name="botaoGerarAVS">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Gerar AVS</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="2">
        <widget class="QPushButton" name="botaoPreview">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Pré-Visualizar</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="5" alignment="Qt::AlignTop">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="ativarEdLegendas">
            <property name="text">
             <string>Ativar Legendas</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ativarEdComentarios">
            <property name="text">
             <string>Ativar comentários</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ativarCorte">
            <property name="text">
             <string>Ativar corte</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>659</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionNovo"/>
    <addaction name="actionSalvar"/>
    <addaction name="actionSalvar_Como"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuOp_es">
    <property name="title">
     <string>Opções</string>
    </property>
    <addaction name="actionEditar_Legendas"/>
    <addaction name="actionEditar_Video"/>
    <addaction name="actionEditar_Coment_rios"/>
    <addaction name="actionCorte"/>
    <addaction name="actionPr_Visualizar"/>
    <addaction name="separator"/>
    <addaction name="actionPrefer_ncias"/>
   </widget>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionManual"/>
    <addaction name="actionAtualiza_es"/>
    <addaction name="actionSobre"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuOp_es"/>
   <addaction name="menuAjuda"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>24</height>
    </size>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNovo"/>
   <addaction name="actionSalvar"/>
   <addaction name="separator"/>
   <addaction name="actionEditar_Legendas"/>
   <addaction name="actionEditar_Video"/>
   <addaction name="actionEditar_Coment_rios"/>
   <addaction name="actionCorte"/>
   <addaction name="actionPr_Visualizar"/>
   <addaction name="separator"/>
   <addaction name="actionPrefer_ncias"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionNovo">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/Icones/novo.svg</normaloff>:/Icones/novo.svg</iconset>
   </property>
   <property name="text">
    <string>Novo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/Icones/salvar.svg</normaloff>:/Icones/salvar.svg</iconset>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSalvar_Como">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/Icones/salvarcomo.svg</normaloff>:/Icones/salvarcomo.svg</iconset>
   </property>
   <property name="text">
    <string>Salvar Como</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionEditar_Legendas">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/Icones/legenda.svg</normaloff>:/Icones/legenda.svg</iconset>
   </property>
   <property name="text">
    <string>Editar Legendas</string>
   </property>
  </action>
  <action name="actionEditar_Video">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/Icones/video.svg</normaloff>:/Icones/video.svg</iconset>
   </property>
   <property name="text">
    <string>Editar Vídeo</string>
   </property>
  </action>
  <action name="actionEditar_Coment_rios">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/Icones/comentario.svg</normaloff>:/Icones/comentario.svg</iconset>
   </property>
   <property name="text">
    <string>Editar Comentários</string>
   </property>
  </action>
  <action name="actionPrefer_ncias">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/Icones/config.svg</normaloff>:/Icones/config.svg</iconset>
   </property>
   <property name="text">
    <string>Preferências</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/Icones/info.svg</normaloff>:/Icones/info.svg</iconset>
   </property>
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionCorte">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/Icones/corte.svg</normaloff>:/Icones/corte.svg</iconset>
   </property>
   <property name="text">
    <string>Corte</string>
   </property>
  </action>
  <action name="actionPr_Visualizar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/Icones/preview.svg</normaloff>:/Icones/preview.svg</iconset>
   </property>
   <property name="text">
    <string>Pré-Visualizar</string>
   </property>
  </action>
  <action name="actionAtualiza_es">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/Icones/atualizar.svg</normaloff>:/Icones/atualizar.svg</iconset>
   </property>
   <property name="text">
    <string>Atualizações</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/Icones/manual.svg</normaloff>:/Icones/manual.svg</iconset>
   </property>
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/Icones/sair.svg</normaloff>:/Icones/sair.svg</iconset>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imgs.qrc"/>
 </resources>
 <connections/>
</ui>
